packages/manager/modules/pci/src/projects/project/kubernetes/details/service/translations/Messages_fr_FR.json

```diff
@@ -27,20 +28,23 @@
   "kube_service_billing_contact": "Nic Billing",
   "kube_service_billing_renewal": "Renouvellement",
   "kube_service_billing_renewal_automatic": "Renouvellement automatique",
-  "kube_service_upgrade_policy": "Politique de mise à niveau",
+  "kube_service_upgrade_policy": "Politique de mise à jour de sécurité",
   "kube_service_upgrade_policy_NEVER_UPDATE": "Ne pas mettre à jour",
   "kube_service_upgrade_policy_MINIMAL_DOWNTIME": "Indisponibilité minimale",
   "kube_service_upgrade_policy_ALWAYS_UPDATE": "Sécurité Maximale",
   "kube_service_upgrade_policy_description_NEVER_UPDATE": "Nous vous déconseillons ce choix. OVH se réserve la possibilité de mettre à jour exceptionnellement des composants Kubernetes ou vos nœuds, dans des cas critiques limitant la sécurité de notre infrastructure.",
-  "kube_service_upgrade_policy_description_MINIMAL_DOWNTIME": "Mettre à jour les composants versions mineures de mon service Kubernetes, pour garantir la sécurité et stabilité du service. Si nous ne pouvons effectuer une mise à jour roulante de vos nœuds sans arrêt de service, celle-ci est reportée. Nous vous conseillons de dimensionner votre cluster pour garantir une mise à jour à tout moment.",
-  "kube_service_upgrade_policy_description_ALWAYS_UPDATE": "Mettre à jour les composants versions mineures de mon service Kubernetes, pour garantir la sécurité et stabilité du service. La mise à jour entrainera possiblement une indisponibilité de quelques minutes lors de la mise à jour roulante de vos nœuds.",
-  "kube_service_upgrade_policy_model_title": "Modifier la politique de mise à niveau",
+  "kube_service_upgrade_policy_description_MINIMAL_DOWNTIME": "Appliquer les mises à jour de sécurité ('patch version') de mon service Kubernetes, pour garantir la sécurité et stabilité du service. Si nous ne pouvons effectuer une mise à jour roulante de vos nœuds sans arrêt de service, celle-ci est reportée. Nous vous conseillons de dimensionner votre cluster pour garantir une mise à jour à tout moment.",
```

Comment:
You should ask the reporter to rewrite this text, it's badly written

Reply:
@ganeshkumar1989 have you asked correct text to Content team ?

Reply:
I have already requested for the content to be corrected by the content team @JeremyDec

Reply:
Thanks Ganesh !